app/models/opensearch.rb

```diff
@@ -105,6 +105,33 @@ def multisearch
     ]
   end
 
+  # https://opensearch.org/docs/latest/query-dsl/minimum-should-match/#valid-values
+  # checks for preconfigured cases or uses whatever is supplied (i.e. we currently accept OpenSearch syntax for
+  # minimum_should_match)
+  def minimum_should_match
```

Comment:
I'm open to holding off on merging until we decide what we want here fwiw. I just needed something in place for stakeholder feedback.

Reply:
That's an interesting idea. As noted below, I have some concerns about introducing/documenting experimental features in the API, since we may remove one or more of them. It's a fairly mild concern, though, since I don't think anyone but us is going to be using these before we decide what we want to do with them.

Reply:
I opted for just documented AND and OR and merging as that'll leave the timdex-ui experimental PR build working for stakeholders (the API still shuts down PRs after a day). I apologize to our future selves when we forget to cleanup these experiments and can't remember why they are there and aren't documented, etc.